rust/src/interpreter/stack.rs

```diff
@@ -0,0 +1,149 @@
+use evmc_vm::{StatusCode, StepStatusCode};
+
+use crate::types::u256;
+
+#[derive(Debug)]
+pub struct Stack(Vec<u256>);
+
+impl Stack {
+    pub fn from(inner: Vec<u256>) -> Self {
+        Self(inner)
+    }
+
+    pub fn empty() -> Self {
```

Comment:
nit: call it new, make it first function in list.

Reply:
Agreed.